src/Microsoft/Silk.NET.Direct2D/WindowsCodecs/Structs/Rect.gen.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Silk.NET.Core;
+using Silk.NET.Core.Native;
+using Silk.NET.Core.Attributes;
+using Silk.NET.Core.Contexts;
+using Silk.NET.Core.Loader;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.WindowsCodecs
+{
+    [NativeName("Name", "WICRect")]
+    public unsafe partial struct Rect
+    {
+        public Rect
+        (
+            int? x = null,
+            int? y = null,
+            int? width = null,
+            int? height = null
+        ) : this()
```

Comment:
Sorry, missed this in the initial review.

Can we map this to `Silk.NET.Maths.Rectangle<int>` and add this type to the `exclude` list? We generally try to avoid duplicating numeric-like structures to ensure maximum portability and interoperability between our APIs.